manifests/istio-telemetry/kiali/values.yaml

```diff
@@ -1,16 +1,14 @@
-#
-# addon kiali
-#
 kiali:
-  enabled: false # Note that if using the demo or demo-auth yaml when installing via Helm, this default will be `true`.
-  replicaCount: 1
-  hub: quay.io/kiali
-  tag: v1.9
-  image: kiali
-  contextPath: /kiali # The root context path to access the Kiali UI.
+
+  enabled: false # If using demo profile, this will be set to `true`.
```

Comment:
I am a bit confused why we have a kiali/ and kiali-operator/

I think that means users would need to do something like `kialiOperator.enabled=true kiali.enabled=true` which doesn't make too much sense to me -- you always want them together, right?

Reply:
You do not necessarily always want both. For example, someone could be installing operators external to istioctl (people using [OLM](https://github.com/operator-framework/operator-lifecycle-manager) / [OperatorHub.io](https://operatorhub.io/) will be installing operators with that mechanism, not directly via istioctl). If someone already installed the Kiali operator, then they need only tell istioctl to install the Kiali CR (`kiali.enabled=true`).

Even if someone installed Kiali Operator via istioctl, there could be (if not now, in the future) a multi-tenant scenario where I want to install multiple instances of Kiali. In this case, you do not want to install Kiali Operator multiple times (you only need a single operator per cluster) - this is another use-case where istioctl is told kialiOperator.enabled=false but kiali.enabled=true